src/phoneField/style.ts

```diff
@@ -0,0 +1,31 @@
+import css from 'styled-jsx/css'
+import { color, radius, space } from '_utils/branding'
+
+export default css`
+  :glonal(.kirk-phoneField--inline) {
```

Comment:
glonal ??

Reply:
Funny `:global` and `:glonal` works. `:nico` doesn't.

Reply:
Done.